include/mf_nh_operator.h

```diff
@@ -936,7 +937,7 @@ NeoHookOperator<dim, fe_degree, n_q_points_1d, Number>::do_operation_on_cell(
             FEEvaluation<dim, fe_degree, n_q_points_1d, dim, Number> phi_grad(
               phi);
             VectorizedArrayType *ref_grads = phi_grad.begin_gradients();
-#if DEAL_II_VERSION_GTE(9, 3, 0)
+
             dealii::internal::FEEvaluationImplCollocation<
               dim,
               n_q_points_1d - 1,
```

Comment:
@peterrum I was just wondering if I can switch to your pre-compiled FEEvaluation instantiations. May I assume that the internal function is a showstopper at this point?

Reply:
What is your goal here? I guess its faster if you explain. Btw. what do you mean by *showstopper*.

Reply:
I want to use `FEEvaluation<dim,-1,0,...>` so that I don't need to compile all the kernels manually again and again, but there is probably no version of `dealii::internal::FEEvaluationImplCollocation<dim, -1` allowing to drop the template argument here?!
Related changes are not yet in this PR.

> Btw. what do you mean by showstopper.

https://de.wiktionary.org/wiki/Showstopper ... maybe it was not clean English :sweat_smile:

Reply:
I guess you could use: https://github.com/dealii/dealii/blob/fcc0cac3761d0964e320540e83573ee213353c5e/include/deal.II/matrix_free/evaluation_template_factory.h#L40

But why aren't you setting up Matrixfree with an additional set of quadrature points?

Reply:
> I guess you could use: https://github.com/dealii/dealii/blob/fcc0cac3761d0964e320540e83573ee213353c5e/include/deal.II/matrix_free/evaluation_template_factory.h#L40

The n_qpoints argument is now gone completely?! Is it really a collocation? And is it the slow path or the fast path?

> But why aren't you setting up Matrixfree with an additional set of quadrature points?

Sounds like a viable alternative. Although this might render potential mapping updates more expensive since MF loops internally over all quadrature formulas IIRC?!

Reply:
> The n_qpoints argument is now gone completely?! Is it really a collocation? And is it the slow path or the fast path?

I guess it depends on the the `ShapeInfo` you hand in. Collection should be fast. But there is function to check it.

Reply:
Alright, thanks!